Content.Client/Audio/Jukebox/JukeboxBoundUserInterface.cs

```diff
@@ -72,7 +73,14 @@ public void Reload()
 
     public void PopulateMusic()
     {
-        _menu?.Populate(_protoManager.EnumeratePrototypes<JukeboxPrototype>().OrderBy(x => x.Name).ToList()); //imp edit - order the list alphabetically
+        //imp edit -- support emagging the jukebox
+        var songList = _protoManager.EnumeratePrototypes<JukeboxPrototype>().Where(x => !x.EmagOnly).ToList();
+        var emagSongList = _protoManager.EnumeratePrototypes<JukeboxPrototype>().Where(x => x.EmagOnly).ToList();
+
+        if (EntMan.TryGetComponent(Owner, out EmaggedComponent? emComp)){ // if the jukebox this is attached to is emagged...
+            songList.AddRange(emagSongList);
+        }
+        _menu?.Populate(songList.OrderBy(x => x.Name)); //imp edit - order the list alphabetically
```

Comment:
nitpick but the whole method is an imp edit now, so this comment probably isnt too necessary